src/AgGrid/ValueFormat.elm

```diff
@@ -0,0 +1,205 @@
+module AgGrid.ValueFormat exposing
+    ( currencyValueFormatter, decimalValueFormatter, percentValueFormatter
+    , numberValueGetter
+    , currencyFilterValueGetter, decimalFilterValueGetter, percentFilterValueGetter
+    )
+
+{-| Formatting of values in the AgGrid table.
+
+Allows to format values as currencies, decimal, and percent.
+
+
+# valueFormatter
+
+@docs currencyValueFormatter, decimalValueFormatter, percentValueFormatter
+
+
+# ValueGetter
+
+@docs numberValueGetter
+
+
+# filterValueGetter
+
+@docs currencyFilterValueGetter, decimalFilterValueGetter, percentFilterValueGetter
+
+-}
+
+import String.Interpolate
+
+
+
+-- VALUE FORMATTER
+
+
+{-| Format a cell value as localized CURRENCY value utilizing the Javascript `Intl.NumberFormat` class.
+
+    cellValue = "1200.35"
+
+    currencyValueFormatter { currency = "EUR", countryCode = "de-DE" }
+    > "1.200,35 €"
+
+    currencyValueFormatter { currency = "EUR", countryCode = "en-US" }
+    > "€1,200.35"
+
+-}
+currencyValueFormatter : { currency : String, countryCode : String } -> String
```

Comment:
Could/should we also allow passing in a float?

Reply:
Do you mean that the `renderer` on the columnDef should accept floats as input?